bin/miden-cli/src/config.rs

```diff
@@ -24,7 +24,7 @@ pub struct CliConfig {
     pub store_filepath: PathBuf,
     /// Path to the directory that contains the secret key files.
     pub secret_keys_directory: PathBuf,
-    /// Address of the Miden node to connect to.
+    /// Address of the default account to use.
```

Comment:
We could give a little more context on how the default account is used. Maybe something like:

```suggestion
    /// Default address to be used when executing transaction in case none is provided.
```